web/nclsite/settings.py

```diff
@@ -77,8 +76,12 @@
 
 DATABASES = {
     'default': {
-        'ENGINE': 'django.db.backends.sqlite3',
-        'NAME': os.path.join(BASE_DIR, 'db.sqlite3'),
+        'ENGINE': 'django.db.backends.mysql',
```

Comment:
@Imran2923 can you check with @jannemartini ... because Martini gave me another value as mentioned here https://github.com/nus-ncl/CTFPro/pull/16/files#diff-4064389f8f6a8fa597a4b180604d57d99a920a9c04be2e7cdda25fcb1ca4d846R71